vassal-app/src/main/java/VASSAL/tools/ErrorDialog.java

```diff
@@ -332,20 +332,18 @@ public static void bug(final Throwable thrown) {
     );
   }
 
-  private static final Set<String> reportedDataErrors =
+  private static final Set<String> reporteddataWarnings =
```

Comment:
`reportedDataWarnings`

Reply:
oops

Reply:
a global-search-and-replace gone slightly astray. Fortunately not MUCH astray :)